docs/TFM/projectTool/Walkthrough.md

```diff
@@ -1,35 +1,57 @@
 # Walkthrough - tool-chain tutorial
 ## Download a text editor  
 - vscode: https://code.visualstudio.com/download  
-- add useful extensions (python, SystemVerilog, venus Terminal, Copilot, Diff, Spell Checker)  
+- add useful extensions (python, SystemVerilog, venus Terminal, Copilot, Diff, Spell Checker) 
 
-## Download gitbash for Windows  
-https://gitforwindows.org/  
+## Download Python  
+- https://www.python.org/downloads/
+
+## Download gitbash for Windows
+Git Bash is a command-line tool designed for Windows users, offering a Unix-like environment. It seamlessly integrates the Git version control system with a Bash shell, providing developers with a powerful and familiar command-line interface.
+- https://gitforwindows.org/  
   
 ## Set gitbash in the vscode  
-- you may configure the ~/.bashrc & ~/.aliases with your preferences.
+- you may configure the ~/.bashrc & ~/.aliases with your preferences. To accomplish this, you may need to create a .bashrc file. Do that by typing the following commands
+```
+touch ~/.bashrc # creating .bashrc file
+code ~/.bashrc # open file in vs code for editing
+```
+- You can also explore an alternative option that doesn't require editing the ~/.bashrc file. In Visual Studio Code, navigate to "Terminal -> New Terminal," and once the terminal is open, click on the "plus" icon in the terminal window and choose the "Git Bash" option.
```

Comment:
Not sure what you mean by this.
I'm not aware of a way to edit the gitbash terminal preferences from vscode